packages/boot/src/booters/service.booter.ts

```diff
@@ -77,8 +69,20 @@ export const ServiceDefaults: ArtifactOptions = {
   nested: true,
 };
 
-function isServiceProvider(cls: Constructor<{}>): cls is ServiceProviderClass {
+function isServiceProvider(cls: Constructor<unknown>) {
   const hasSupportedName = cls.name.endsWith('Provider');
-  const hasValueMethod = 'value' in cls.prototype;
+  const hasValueMethod = typeof cls.prototype.value === 'function';
   return hasSupportedName && hasValueMethod;
 }
+
+function isBindableClass(cls: Constructor<unknown>) {
+  if (MetadataInspector.getClassMetadata(BINDING_METADATA_KEY, cls)) {
+    return true;
+  }
+  if (isServiceProvider(cls)) {
+    debug('Provider class found: %s', cls.name);
+    return true;
+  }
+  debug('Skip class not decorated with @bind: %s', cls.name);
```

Comment:
I guess it is customary to only output informational messages when debug is turned on?

Reply:
Yes. It's not severe enough to use `console.warn/error()`